core/silkworm/common/endian_test.cpp

```diff
@@ -121,8 +121,9 @@ TEST_CASE("Block as key and compact form") {
         auto block_number_compact_bytes{to_big_compact(block_number)};
         CHECK(to_hex(block_number_compact_bytes) == "5485ffde");
         // Convert back and check
-        auto block_number_from_compact{from_big_compact<uint64_t>(block_number_compact_bytes)};
-        CHECK(block_number_from_compact == block_number);
+        uint64_t out64;
```

Comment:
Shouldn't be initialized regardless ?

Reply:
Done in https://github.com/torquem-ch/silkworm/pull/582/commits/d0892f5083faca4051ef738e2d3876a928f7e602